content/en/blog/_posts/2022-09-26-csi-migration-status.md

```diff
@@ -0,0 +1,111 @@
+The `scaleio`, `flocker`, `quobyte` and `storageos` drivers were removed; the others are deprecated and will be removed from core Kubernetes in the coming releases.
+
+| Driver    | Deprecated | Code Removal  |
+| --------- | ---------- | ------------- |
+| Flocker   | 1.22       | 1.25          |
+| GlusterFS | 1.25       | 1.26 (Target) |
+| Quobyte   | 1.22       | 1.25          |
+| ScaleIO   | 1.16       | 1.22          |
+| StorageOS | 1.22       | 1.25          |
+
+## What does it mean for the core CSI Migration feature to go GA?
+
+Core CSI Migration goes to GA means that the general framework, core library and API for CSI migration is 
+stable for Kubernetes v1.25 and will be part of future Kubernetes releases as well.
+
+- If you are a Kubernetes distribution maintainer, this means if you disabled `CSIMigration` feature gate previously, you are no longer allowed to do so because the feature gate has been locked.
+- If you are a Kubernetes storage driver developer, this means you can expect no backwards incompatibility changes in the CSI migration library.
+- If you are a Kubernetes maintainer, expect nothing changes from your day to day development flows.
+- If you are a Kubernetes user, expect nothing changes from your day to day usage flows. If you encounter any storage related issues, contact the people who operate your cluster (if that's you, contact the provider of your Kubernetes distribution, or get help from the [community](/community/#discuss)).
+
+## What does it mean for the storage driver CSI migration to go GA?
+
+Storage Driver CSI Migration goes to GA means that the specific storage driver supports CSI Migration. Expect feature parity between the in-tree plugin with the CSI driver.
+
+- If you are a Kubernetes distribution maintainer, make sure you install the corresponding
+CSI driver on the distribution. And make sure you are not disabling the specific `CSIMigration{provider}` flag as they are locked.
+- If you are a Kubernetes storage driver maintainer, make sure the CSI driver can ensure feature parity if it supports CSI migration.
+- If you are a Kubernetes maintainer/developer, expect nothing changes from your day to day development flows.
+- If you are a Kubernetes user, the CSI Migration feature should be completely transparent
+to you, the only requirement is to install the corresponding CSI driver.
+
+## What's next?
+
+We are expecting cloud provider in-tree storage plugins code removal to start to happen as part of the v1.26 and v1.27 releases of Kubernetes. More and more drivers that support CSI migration will go GA in the upcoming releases.
+
+## How do I get involved?
+
+The Kubernetes Slack channel [#csi-migration](https://kubernetes.slack.com/messages/csi-migration) along with any of the standard [SIG Storage communication channels](https://github.com/kubernetes/community/blob/master/sig-storage/README.md#contact) are great mediums to reach out to the SIG Storage and migration working group teams.
```

Comment:
```suggestion
The Kubernetes Slack channel [#csi-migration](https://kubernetes.slack.com/messages/csi-migration) along with any of the standard [SIG Storage communication channels](https://github.com/kubernetes/community/blob/master/sig-storage/README.md#contact) are great media to reach out to the SIG Storage and migration working group teams.
```
I've always been told that the plural of "medium" is "media". Sounds a bit weird here, though. Maybe you could just say "venues" or "places".

Reply:
Mediums are what you run out of first in a clothing retailer. They're also people you visit if you want to take part in [cold reading](https://en.wikipedia.org/wiki/Cold_reading).

I'd write “ways”, and **definitely** not “media”.

Reply:
Done